src/tracing/trace.test.ts

```diff
@@ -11,10 +12,9 @@ jest.mock('@sentry/react', () => {
     startTransaction: jest.fn(),
   }
 })
-const startTransaction = Sentry.startTransaction as jest.Mock
 
 function getTransaction(index = 0): Transaction {
-  const transactions = startTransaction.mock.results.map(({ value }) => value)
+  const transactions = mocked(Sentry.startTransaction).mock.results.map(({ value }) => value)
```

Comment:
it is a little strange to me that this function has side effect on the mock that requires a reset in between tests. like could it break tests if you call it twice in one test? but don't need to fix it as part of this PR